tools/wpt/browser.py

```diff
@@ -396,14 +417,16 @@ class Edge(Browser):
     def install(self, dest=None):
         raise NotImplementedError
 
+    def find_binary(self):
+        raise NotImplementedError
```

Comment:
Comment? Or maybe hoist all the comments into one place and remove them above instead.

Reply:
Let me put some texts in the base class.